pkg/generate/git/list_remote_timeout_test.go

```diff
@@ -0,0 +1,47 @@
+package git
```

Comment:
just a nit, but to follow convention, shouldn't this be called repository_test.go?